src/tpt_reports/tpt_reports.py

```diff
@@ -171,11 +172,13 @@ def main() -> None:
     output_directory: str = validated_args["OUTPUT_DIRECTORY"]
     json_file_path: str = validated_args["JSON_FILE_PATH"]
 
-    # Issue #40 - Add additional logging configuration
-    # TODO: Update basicConfig to append logs to an external file.
     # Set up logging
     logging.basicConfig(
-        format="%(asctime)-15s %(levelname)s %(message)s", level=log_level.upper()
+        filename=LOGGING_FILE,
+        filemode="a",
+        format="%(asctime)s - %(name)s - %(levelname)s - %(message)s",
+        datefmt="%m/%d/%Y %I:%M:%S",
+        level=log_level.upper(),
```

Comment:
Alphabetize by parameter name:
```suggestion
        datefmt="%m/%d/%Y %I:%M:%S",
        filemode="a",
        filename=LOGGING_FILE,
        format="%(asctime)s - %(name)s - %(levelname)s - %(message)s",
        level=log_level.upper(),
```